poradnia/letters/templates/letters/email/letter_created.txt

```diff
@@ -1,5 +1,6 @@
+{% load format_text%}
```

Comment:
Wygląda na to, że ta zmiana wywołała problem #930 , bo wysyłamy jako temat pierwszą linijkę https://github.com/watchdogpolska/poradnia/blob/a56a3df64469019073302f8abee558fab32af922/poradnia/template_mail/utils.py#L120 . Myślę, że potrzebujemy w tym miejscu trimować tekst, a także dodać adekwatne testy.